fuse_optimizers/src/optimizer.cpp

```diff
@@ -344,6 +365,25 @@ void Optimizer::stopPlugins()
   {
     name_plugin.second->stop();
   }
+
+  diagnostic_updater_.force_update();
+}
+
+void Optimizer::setDiagnostics(diagnostic_updater::DiagnosticStatusWrapper& status)
+{
+  if (!ros::Time::isValid())
```

Comment:
BTW I'm not 100% sure this is needed. If the `ros::Time` is not valid yet, the timer won't trigger either. I think it doesn't hurt to have it, but I'm OK if you want to remove it.